test/unit/groupTest.js

```diff
@@ -0,0 +1,116 @@
+const helpers = require ('../helpers/createDanceAPI');
+const test = require('tape');
+
+test('changing dance moves for all updates all dancers', async t => {
+  const nativeAPI = await helpers.createDanceAPI();
+  nativeAPI.play({
+    bpm: 120,
+  });
+  nativeAPI.setAnimationSpriteSheet("CAT", 0, {}, ()=> {} );
+  nativeAPI.setAnimationSpriteSheet("CAT", 1, {}, ()=> {} );
+  nativeAPI.setAnimationSpriteSheet("BEAR", 0, {}, ()=> {} );
+  nativeAPI.setAnimationSpriteSheet("BEAR", 1, {}, ()=> {} );
+
+  const catSprite = nativeAPI.makeNewDanceSprite("CAT", null, {x: 200, y: 200});
+  const bearSprite = nativeAPI.makeNewDanceSprite("BEAR", null, {x: 200, y: 200});
+
+
+  t.equal(catSprite.current_move, 0);
+  t.equal(bearSprite.current_move, 0);
+  nativeAPI.changeMoveEachLR('all', 1);
+
+  t.equal(catSprite.current_move, 1);
+  t.equal(bearSprite.current_move, 1);
```

Comment:
Noticing that we're not consistent about camelCase in the API — wonder if we should update everywhere?  Not blocking.